src/cli.rs

```diff
@@ -28,11 +71,21 @@ pub fn cli_init() -> AppConfig {
                 .help("Input file containing links")
                 .takes_value(true),
         )
+        .arg(
+            Arg::with_name("output_directory")
+                .long("output_directory")
+                .short("o")
+                .help("Directory for store output epub documents")
+                .conflicts_with("output_name")
+                .long_help("Directory for saving epub documents")
```

Comment:
The `long_help` is not needed since the help message above is descriptive enough.